microprofile/WFLY-19592_MicroProfile_Fault_Tolerance_4_1.adoc

```diff
@@ -0,0 +1,127 @@
+---
+categories:
+- microprofile
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/626
+feature-team:
+ developer: rhusar
+ sme:
+  - rhusar
+ outside-perspective:
+  -
+promotes:
+promoted-by:
+---
+= WFLY-19592 [MP Platform 7] MicroProfile Fault Tolerance 4.1
+:author:            Radoslav Husar
+:email:             rhusar@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Goal is to integrate MicroProfile (MP) Fault Tolerance (FT) 4.1 specification from the MP Platform 7.0 to WildFly at default stability level.
+To certify functionality, the MP FT TCK is run as part of the WildFly testsuite.
+
+=== User Stories
+
+User wants to deploy an application using MicroProfile Fault Tolerance 4.1 from MicroProfile Platform 7.0.
+
+== Issue Metadata
+
+* https://issues.redhat.com/browse/WFLY-19592[WFLY-19592 [MP Platform 7] MicroProfile Fault Tolerance 4.1]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-19592[WFLY-19588 Implement MicroProfile Platform 7] – MP Platform parent issue
+* https://issues.redhat.com/browse/WFLY-19590[WFLY-19590 MicroProfile Telemetry 2.0] – dependency of this issue for full Telemetry metrics integration
+
+// EAP Tracking issue:
+// * https://issues.redhat.com/browse/EAP7-2225[EAP7-2225 [MP Platform 7] MicroProfile Fault Tolerance 4.1]
+
+=== Affected Projects or Components
+
+* Eclipse MicroProfile Fault Tolerance
+* SmallRye Fault Tolerance
+
+=== Other Interested Projects
+
+* N/A
+
+=== Relevant Installation Types
```

Comment:
A question I've asked when reviewing the parent tracker as well, mainly due to similar considerations we've done when adding/upgrading the MP specs integration in the past, is whether the "managed domain" would be relevant as well. 
We all know MicroProfile is about standalone instances (and should be about single deployments too, although this is a whole different topic) but I recall one discussion in which it was marked as relevant since in theory nothing would prevent a configuration from having the extension enabled.
As said, I don't have a strong opinion, while this is rather something I'd like to discuss for the sake of clarity and common approach to similar issues, now and in the future.

Reply:
I used what all other MicroProfile specifications are declaring, so this needs to be discussed globally if a change is needed. But yes, technically nothing preventing a managed domain, but that's not my call.

Reply:
@fabiobrz FYI Added back managed domain as per my last comment.

Reply:
Thanks @rhusar !